calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -0,0 +1,61 @@
+---
+description: Identify vulnerabilities in Kubernetes clusters.
+---
+
+# Automatically scan using cluster Scanner
+
+## Big picture
+
+Automatically scan kubernetes workloads for vulnerabilities using {{prodname}} Image Assurance Cluster Scanner.
+
+
+## Before you begin
+
+**Cluster requirements**
+
+To be included in scan results, cluster must be:
+
+- Using containerd as the container runtime.
+- Containerd must be using overlayfs or native file system snapshotter.
+
+**Image Requirements**
+- Workloads must be running an image that has RepoDigest ID
+
+## How to
+
+- [Enable cluster scanner](#enable-the-cluster-scanner)
+- [Configure cluster scanner](#configure-cluster-scanner)
+- [Disable cluster scanner](#disable-cluster-scanner)
+
+### Enable the cluster scanner
+
+1. In your cluster, enable the cluster scanner by modifying the `imageassurances` custom resource.
+
+   ```shell
+      kubectl edit imageassurances default
+   ```
+
+2. Set the `clusterScanner` field to `Enabled` and save the file.
+
+3. This will deploy the cluster scanner as a daemon set in your cluster. The scanner is deployed as a container inside `tigera-image-assurance-crawdad`
```

Comment:
change all occurrences of "daemon set" to "daemonset".

Reply:
fixed with bed5e89